docs/source/debugging.rst

```diff
@@ -189,9 +190,48 @@ Defaults currently look like the following:
      distributed.client: warning
      bokeh: error
 
-So, for example, you could add a line like ``distributed.worker: debug`` to get
+The specific components which you can set are ``distributed.client``,  ``distributed.scheduler``,
+``distributed.nanny``, and ``distributed.worker`` so, for example, you could add a line
+like ``distributed.worker: debug`` to get
 *very* verbose output from the workers.
 
+Furthermore, you can explicitly assign handlers to loggers. The following example
+assigns both file ("output.log") and console output to the scheduler and workers; this case is done
+in code. See the `python logging`_ documentation for information on the meaning of
+specific terms here.
+
+.. code-block:: python
```

Comment:
Thoughts on converting this to YAML and writing the example as if you're in the dask-config?

Otherwise we should have a strong warning that the changes here need to be applied before any workers are created (and perhaps before distributed is even imported. I don't recall what all is done at import time).

Reply:
It would be nice to have both, but that would be rather long. I don't think there's an example of complex configuration by code elsewhere ( @quasiben ?)

Reply:
To be clear, I'd say just have the YAML version.


```yaml
# ~/.config/dask/dask.yaml
logging:
  version: 1
  handlers:
    file:
      class: logging.handlers.RotatingFileHandler
...
```

I *think* that should work.

Reply:
What is the `version: 1`, by the way?

Reply:
It's required by Python's logging config: https://docs.python.org/3/library/logging.config.html#dictionary-schema-details